rest/handler.go

```diff
@@ -725,26 +725,35 @@ func (h *handler) checkAuth(dbCtx *db.DatabaseContext) (err error) {
 		return nil
 	}
 
+	auditFields := make(base.AuditFields)
```

Comment:
As discussed offline, I'm worried about the note at the top of this function: `// NOTE: checkAuth is not used for the admin interface.` and might want to see this in common code.

Reply:
Acknowledged. I'll see if I can generalise/clean up the code around these two events.

Reply:
I partially solved this by choosing to have separate audit events for public API users and admin API users, which makes more sense anyway.

I'd prematurely implemented these events to put that demo together without thinking through the actual auditing use-cases.